molecule/default/requirements.yml

```diff
@@ -23,10 +23,14 @@
   src: https://github.com/onaio/ansible-gpg-import.git
   scm: git
 
+- src: https://github.com/DavisRayM/python.git
+  name: DavisRayM.python
+  scm: git
+
```

Comment:
I don't think this is needed. The `onaio.django` dependency now describes where to get this package, molecule should install it when it's testing.

_[ansible-django change](https://github.com/onaio/ansible-django/blob/master/meta/main.yml#L13-L24)_

Reply:
ah..sawa...al remove it